spec/features/post_show_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'user_index', type: :feature do
+  user = User.first
+  post = user.posts.first
+  before(:each) do
+    visit(user_post_path(user.id, post.id))
+  end
+
+  it 'shows the title of the post' do
+    expect(page).to have_content(post.title)
+  end
```

Comment:
- [ ] The same thing with the posts_show page, try to create a user and a post then add feature tests for the posts_show page. Your tests fail because there are no users in your DB:

<img width="399" alt="image" src="https://user-images.githubusercontent.com/77190203/187890743-f0b2f3da-3835-4ef7-866e-8a54a28fae57.png">

Don't forget to create some comments and likes so you are able to test them as well.

Reply:
Done ✔